packages/utils/src/types.ts

```diff
@@ -358,6 +358,8 @@ export type FieldTemplateProps<
   id: string;
   /** A string containing the base CSS classes, merged with any custom ones defined in your uiSchema */
   classNames?: string;
+  /** An object containing the styles as defined in the `uiSchema` */
+  styles?: StyleHTMLAttributes<any>;
```

Comment:
It seemed like the preference in the original PR comments was `style` rather than `styles`, which would match the React attribute name